packages/sdk/src/test/methods.spec.ts

```diff
@@ -349,20 +374,20 @@ describe('LookerNodeSDK', () => {
       expect(sdk.authSession.isAuthenticated()).toBeFalsy()
     })
 
-    it('search_looks fields filter', async () => {
+    it('search_looks fields and title', async () => {
       const sdk = new LookerSDK(session)
       const actual = await sdk.ok(
         sdk.search_looks({
           fields: 'id,title',
-          title: 'Order%',
+          title: 'An SDK%',
         })
       )
       expect(actual).toBeDefined()
-      expect(actual.length).toBeGreaterThan(1)
+      expect(actual.length).toBeGreaterThanOrEqual(1)
```

Comment:
I'm sure this is correct but to be greater than or equal to 1? It's not clear why

Reply:
There could be more than one look beginning with `An SDK`